validator/utils/minio.py

```diff
@@ -53,6 +54,14 @@ async def list_objects(self, bucket_name, prefix=None, recursive=True):
         logger.info("Listing objects")
         return await self.loop.run_in_executor(self.executor, func, *args)
 
+    async def get_stats(self, bucket_name: str, object_name: str):
```

Comment:
Does get_stats work no matter what's inside? So csv, json, parquet no issues?

Reply:
yes it should work regardless of the file type since it simply returns metadata like size etc
But I'll test it anyways just to make sure

Reply:
I tested it and it works as expected, it took me a bit of time since I had trouble debugging bcz the minio class has its own event loop and I couldn't run debug stuff outside of it xD